fri/test.py

```diff
@@ -73,13 +70,30 @@ def openJupyter():
   response = requests.request("POST", url)
   print(response.text)
 
+# function to test download() method.
+def download():
+  url = "http://127.0.0.1:5000/download/test?fetch=f1.txt&apikey=xyz"
+  urllib.request.urlretrieve(url, "f1.txt")
 
-# upload()
-# execute()
-# download()
-# build()
-# debug()
-# destroy()
-getFilesList()
+# file list to be uploaded
+files=[
+  #('files[]',(file_name,open(file_path,'rb'),'application/octet-stream'))
+  ('files[]',('controller.py',open('/home/amit/Desktop/test_xyz/controller.py','rb'),'application/octet-stream')),

```

Comment:
Pls do not hard-code the values such as "/home/amit" as these will not work outside your computer. Use relative paths instead.

Reply:
Replace '/home/amit/Desktop/test_xyz/controller.py' with '../demo/controller.py'